test/e2e/flask/btc/common-btc.ts

```diff
@@ -0,0 +1,59 @@
+import { Mockttp } from 'mockttp';
```

Comment:
Nit: But we could have just named it `common.ts` since it's already in `test/e2e/flask/btc/` folder!